integration-cli/docker_cli_kill_test.go

```diff
@@ -26,6 +26,11 @@ func (s *DockerSuite) TestKillContainer(c *check.C) {
 		c.Fatalf("failed to kill container: %s, %v", out, err)
 	}
 
+	/* Container should be stopped, second kill should return an error */
+	if _, _, err = runCommandWithOutput(killCmd); err == nil {
```

Comment:
it might be better to create a new test where you:
- create a container
- stop it
- try to kill it and then check for your new error message.
I think its important to make sure that we run, and verify, that the code you added is hit.